### PR TITLE
Additional fix for error message "Unable to insert the virtual optica…

### PR DESCRIPTION
**Pull request by arktal**
…l disk"

Added a fix that worked for me on line 134 about ticking the "Live CD/DVD" box.

 - [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [ ] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [ ] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [ ] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

...your text here

#### 2. Related Issue

Closes #XXXXX

### CODE REVIEW

File: foundations/installations/prerequisites.md [hunk 1]
```diff
@@ -130,7 +130,9 @@ While your VM is running, do the following steps:
   **NOTE**:
 
 * If upon trying to start the VM you only get a black screen, close and "power off" the VM, click "Settings -> Display" and make sure "Enable 3D Acceleration" is UNCHECKED, and Video memory is set to AT LEAST 128mb.
-* If you receive an error when trying to mount the Guest Additions CD image ("Unable to insert the virtual optical disk"), please reboot your host (Windows/OSX) operating system. Afterwards, ensure that there is no image file mounted in *both* Virtual Box as well as in the file system of the VM.
+* If you receive an error when trying to mount the Guest Additions CD image ("Unable to insert the virtual optical disk"):
+   1. In VirtualBox Manager, select Xubuntu then click Settings. In the Storage tab, under Controller: IDE, click on VBoxGuestAdditions.iso and make sure "Live CD/DVD" is ticked.
+   2. Please reboot your host (Windows/OSX) operating system. Afterwards, ensure that there is no image file mounted in *both* Virtual Box as well as in the file system of the VM.
```
Comment (zachmmeyer) [inline foundations/installations/prerequisites.md:135]:
```suggestion
   Suggestion 2: In VirtualBox Manager, while the VM is not running, select Xubuntu then click Settings. In the Storage tab, under Controller: IDE, click on VBoxGuestAdditions.iso and make sure "Live CD/DVD" is ticked. Enabling this option causes the image to not be removed upon ejection, therefore it should be removed as the final step. To do so, once you have completed the Guest Additions installation and shut down your VM, you can find the image where you enabled "Live CD/DVD": under Controller: IDE, by right-clicking "VBoxGuestAdditions.iso" and click "Remove Attachment". It is also suggested to uncheck "Live CD/DVD" at this point.
```

File: foundations/installations/prerequisites.md [hunk 2]
```diff
@@ -130,7 +130,9 @@ While your VM is running, do the following steps:
   **NOTE**:
 
 * If upon trying to start the VM you only get a black screen, close and "power off" the VM, click "Settings -> Display" and make sure "Enable 3D Acceleration" is UNCHECKED, and Video memory is set to AT LEAST 128mb.
-* If you receive an error when trying to mount the Guest Additions CD image ("Unable to insert the virtual optical disk"), please reboot your host (Windows/OSX) operating system. Afterwards, ensure that there is no image file mounted in *both* Virtual Box as well as in the file system of the VM.
+* If you receive an error when trying to mount the Guest Additions CD image ("Unable to insert the virtual optical disk"):
+   1. In VirtualBox Manager, select Xubuntu then click Settings. In the Storage tab, under Controller: IDE, click on VBoxGuestAdditions.iso and make sure "Live CD/DVD" is ticked.
```
Comment (zachmmeyer) [inline foundations/installations/prerequisites.md:134]:
```suggestion
   Suggestion 1: Reboot your host (Windows/OSX) operating system. Afterward, ensure that there is no image file mounted in *both* Virtual Box as well as in the file system of the VM.
```

Comment (zachmmeyer) [reply foundations/installations/prerequisites.md:134]: We should leave the first suggestion as the simplest.

File: foundations/installations/prerequisites.md (hunk 1 above)
Comment (zachmmeyer) [reply foundations/installations/prerequisites.md:135]: Enabling Live CD/DVD causes a potential problem for the future user who wants to update Guest Additions again if and when they update VirtualBox, therefore more information must be supplied.

**Review by zachmmeyer — CHANGES_REQUESTED** [review]
I'm glad this solution worked for you. There are some changes that need to be made in order for this to work for everybody and not cause issues for the future.

**Review by zachmmeyer — APPROVED** [review]
Thank you very much!

**Comment by arktal** [comment]
> 

Thank you for your suggestion, and sorry for the delay!

**Pull request merged**